docs/apis/plugintypes/tiny/testing.md

```diff
@@ -0,0 +1,181 @@
+Given I set the following fields in the "Create event" "dialogue" to these values:
+  | Name        | Some name here                              |
+  | Description | <p>Some value which includes valid HTML</p> |
+```
+
+##### Checking the content of a field
+
+```gherkin title="Check that the content of the field with the label 'Description' matches a value"
+Then the field "Description" matches value "<p>Some value which includes valid HTML</p>"
+Then the field "Description" matches multiline:
+"""
+<p>Some value which includes valid HTML</p>
+"""
+```
+
+```gherkin title="Check that the content of the field with the label 'Description' does not match a value"
+Then the field "Description" does not match value "<p>Some value which includes valid HTML</p>"
+```
+
+```gherkin title="Check that the content of the field with the label 'Description' within the 'Create event' modal matches a value"
+Then the field "Description" in the "Create event" "dialogue" matches value "<p>Some value which includes valid HTML</p>"
+```
+
+```gherkin title="Check that the content of a set of fields match values"
+Then the following fields match these values:
+  | Name        | Some name here                              |
+  | Description | <p>Some value which includes valid HTML</p> |
+```
+
+```gherkin title="Check that the content of a set of fields within the 'Create event' modal match values"
+Then the following fields in the "Create event" "dialogue" match these values:
+  | Name        | Some name here                              |
+  | Description | <p>Some value which includes valid HTML</p> |
+```
+
+#### TinyMCE specific steps
+
+If you are writing a TinyMCE plugin or feature, then you may need to interact with the editor interface, or to check and set the current value of the editor.
+
+:::note iFrames
+
+Behat makes use of WebDriver to control browsers. The WebDriver specification limits the scope of a command to the current _context_ where a context is a browser Window, Frame, or iFrame.
+
+TinyMCE places the Editor UI within its own iFrame. To control any of its interface, you _may_ need to switch to the iFrame for your editor first.
+
+Most of the steps written for TinyMCE are already aware of the iFrame.
+
+:::
+
+##### Working with toolbar buttons
+
+```gherkin title="Expanding all toolbar buttons"
+Given I expand all toolbars for the "Description" TinyMCE editor
+```
+
+:::caution
+
+The default TinyMCE configuration will automatically collapse the toolbar to show as many elements as will fit on the page.
+
+You should always use the "I expand all toolbars" step before interacting with a button on the toolbar as it may not be visible in all cases.
+
+This step is safe to call if the toolbar is already expanded, or does not need to be expanded.
+
+:::
+
+```gherkin title="Clicking on a button for your editor"
+When /^I click on the "Bold" button for the "Description" TinyMCE editor
```

Comment:
```suggestion
When I click on the "Bold" button for the "Description" TinyMCE editor
```

Reply:
Hi @andrewnicols Thanks for reviewing it. However, it should be changed in versioned_docs/version-4.1/apis/plugintypes/tiny/testing.md too :-P